nemo_text_processing/inverse_text_normalization/es/taggers/date.py

```diff
@@ -53,7 +54,12 @@ def __init__(self):
 
         graph_dm = day_graph + delete_space + pynutil.delete("de") + delete_extra_space + month_graph
 
-        final_graph = graph_dm
+        # transform "siglo diez" -> "siglo x" and "año mil novecientos noventa y ocho" -> "año mcmxcviii"
+        roman_numerals = int_to_roman(cardinal.graph)
+        roman_centuries = pynini.union("siglo ", "año ") + roman_numerals
```

Comment:
I think we should only transform to roman numerals after 'siglo', and not after 'año'.
It is better to do "año mil novecientos noventa y ocho" -> "año 1998" as this is much quicker to read.